pagopa/io-app
# I18n keys sorting

## Short description
sorted all the I18n keys using [`scoped-sort`](https://scopedsort.netlify.app/docs)

## List of changes proposed in this pull request
- sorted ENG locales
- sorted IT locales
- sorted DE locales

## How to test
`yarn generate:locales` followed by `yarn tsc:noemit` should not throw any error, also run the app to make sure nothing broke, although nothing should have
